### PR TITLE
contrib/xstow: new package (1.1.1)

### PR DESCRIPTION
**Pull request by wezm**

### CODE REVIEW

File: contrib/xstow/template.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+pkgname = "xstow"
+pkgver = "1.1.0"
+pkgrel = 0
+build_style = "gnu_configure"
+make_dir = "."
+hostmakedepends = ["automake"]
+makedepends = ["ncurses-devel"]
+pkgdesc = "Extended replacement of GNU Stow"
```
Comment (q66) [inline contrib/xstow/template.py:8]: this description is not very descriptive, it should say what the software does, not what it's a replacement of

**Comment by q66** [comment]
you should also test it with `hardening = ["vis", "cfi"]`

**Comment by wezm** [comment]
Ok I tweaked the description and turned on those hardening options. Still seems to build and run with them enabled

**Comment by wezm** [comment]
I nudged upstream to publish a new release with the clang patches included, which they did. So I've updated the template and dropped the patches.

**Pull request merged**